internal/locationutil/location.go

```diff
@@ -0,0 +1,52 @@
+package locationutil
```

Comment:
I've been looking at this package and I don't think this is what we want to create a clean API. I'm suspecting that it would be easier if, instead of trying to duplicate window code here, we just implemented the logic directly inside of the date package.

It seems to me the core logic is inside of `ToUTCTime` so let's just take that method, let it take a location name, an offset, and the original time, and then have it return a `values.Time` that corresponds to the appropriate translated time. Something like:

```
func getTimeInLocation(t values.Time, location string, offset values.Duration) (values.Time, error)
```

The function can load the location by the name, apply the offset, then return the translated time. We'll avoid this package and then just use the one function we need rather than try to create a general pathway through two areas that aren't very similar beyond using the same struct attributes.

I understand this function will likely be needed in both `date` and `experimental`. We can also put this function in a third `internal/date` package which can be used by both.

Reply:
implemented!